### ISSUE TITLE
Better styling for login button

### ISSUE THREAD

**Issue by filipfigzalski** [9p4/jellyfin-plugin-sso#161]
I was able to figure out how to put the "Sign in with Authentik" button below the normal sign in button. This way users will know they have two options for login.

## The looks
![image](https://github.com/9p4/jellyfin-plugin-sso/assets/61553732/51e12df1-f404-47e3-a718-a9ceecdecc7f)

### Login disclaimer:
```html
<form action="https://jellyfin.example.com/sso/OID/start/PROVIDER_NAME">
<button class="raised block emby-button button-submit">Sign in with PROVIDER_NAME</button>
</form>
```

### Custom CSS code:
```css
.disclaimerContainer {
  display: block;
  margin-top: 0em;
  margin-bottom: 1em;
}

.readOnlyContent {
  display: flex;
  flex-direction: column-reverse;
}
```

**Comment by 9p4** [comment]
Related: https://github.com/9p4/jellyfin-plugin-sso/issues/16

**Comment by 9p4** [comment]
This form-based behaviour is better than the old link-based one. I'm going to switch the README to use this.

**Comment by 9p4** [comment]
Implemented in ec82bf7
